apps/core/src/hooks/affine/use-is-shared-page.ts

```diff
@@ -31,18 +31,14 @@ export function useIsSharedPage(
       (enable: boolean) => {
         // todo: push notification
         if (enable) {
-          enableSharePage({
-            workspaceId,
-            pageId,
-          })
+          enableSharePage({ docId: `${workspaceId}:space:${pageId}` })
```

Comment:
https://github.com/toeverything/blocksuite/pull/4747 carefully bump blocksuite the next time @himself65